### ISSUE TITLE
Screen Reader Accessibility in UI on Windows

### ISSUE THREAD

**Issue by Gamulation** [Ryujinx/Ryujinx#4352]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of Issue

Navigating around with keyboard doesn't work. I am unable to get speech via my screen reader to know where I am in the UI.
This would allow me to use the program as a blind user. Currently there is no speech when launching the program and I am unable to do anything. I use this screen reader here: https://www.nvaccess.org/download/
I really hope this can be implemented.

### Reproduction Steps

Open the program with NVDA or the default windows screen reader and try to navigate with the keyboard or try to navigate with mouse. The result is no speech no matter what you do.

### Log File

No log file

### OS

Windoes 10

### Ryujinx version

Any

### Game version

Ryujinx UI

### Environment?

I7 8700
Nvidia RTX 2070
16GB DDR4 ram

### Additional context?

Any Screen reader will do. The default windows one can be turned on by holding down CTL, windows key and pressing enter. Same goes for turning it off.

**Comment by IsaacMarovitz** [comment]
Are you using GTK or Ava UI?

**Comment by IsaacMarovitz** [comment]
<table>
<tr>
<th>Accessibility</th>
<th>GTK</th>
<th>Avalonia</th>
</tr>
<tr>
<th>Windows</td>
<td>?</td>
<td>?</td>
</tr>
<tr>
<th>macOS</th>
<td>NA</td>
<td>?</td>
</tr>
<tr>
<th>Linux</th>
<td>?</td>
<td>?</td>
</tr>
</table>

**Comment by RealAmethyst** [comment]
Now that Avalonia 11 has released. Would it be possible to implement accessibility API's? This should finally let people who use a screen reader be able to use Ryujinx. I really hope this is something that can be added. Thanks!

**Comment by rings2006** [comment]
just wanted to say here that there are free screen readers for windows to test things with, and even one built in, narrator, its not the best screen reader in the world but perfectly fine for testing this, on linux theres orca and on mac there is voiceover, also theres odilia for linux but its not ready yet
